core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -769,6 +769,12 @@ public static function create($image, $size=null)
 			$size = (int) $size[2];
 		}
 
+		// predefined image size as resize mode
+		elseif (\is_array($size) && !empty($size[2]) && is_string($size[2]) && $size[2][0] === '_')
```

Comment:
There already is an `is_array($size)` check in line 767 and another one in line 778. Does this need to be refactored?

Reply:
Done in e97bba284687fa9c6db236f032eebf2238922d6a